### ISSUE TITLE
swaylock-fancy

### ISSUE THREAD

**Issue by eoli3n** [NixOS/nixpkgs#77284]
**Project description**
Port of i3lock-fancy to swaylock, which takes a screenshot of the desktop, blurs the background and adds a lock icon and text

**Metadata**

* homepage URL: https://github.com/Big-B/swaylock-fancy
* source URL: https://github.com/Big-B/swaylock-fancy
* license: mit https://github.com/Big-B/swaylock-fancy/blob/master/LICENSE
* platforms: linux

**Comment by Ma27** [comment]
Nice, didn't know about that fork! :)
Just added the package to master, in a few days it should be available on nixos-unstable :)

**Comment by eoli3n** [comment]
It's actually not working, can't build it
```
installing 'swaylock-fancy-unstable-2019-03-31'
these derivations will be built:
  /nix/store/rcjqja6g7ndwy261r3z743s28w2lripp-source.drv
  /nix/store/i5h8w2rfq039212d1ayn41m9jiyamfli-swaylock-fancy-unstable-2019-03-31.drv
building '/nix/store/rcjqja6g7ndwy261r3z743s28w2lripp-source.drv'...

trying https://github.com/Big-B/swaylock-fancy-unstable/archive/35618ceec70338047355b6b057825e68f16971b5.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (22) The requested URL returned error: 404 Not Found
error: cannot download source from any mirror
builder for '/nix/store/rcjqja6g7ndwy261r3z743s28w2lripp-source.drv' failed with exit code 1
cannot build derivation '/nix/store/i5h8w2rfq039212d1ayn41m9jiyamfli-swaylock-fancy-unstable-2019-03-31.drv': 1 dependencies couldn't be built
error: build of '/nix/store/i5h8w2rfq039212d1ayn41m9jiyamfli-swaylock-fancy-unstable-2019-03-31.drv' failed
```

**Comment by Ma27** [comment]
Ouch, sorry! Fixed in ec2ed2d316d850061ffa75a43f487928b0d7e7c3.
